plugins/node/opentelemetry-instrumentation-express/test/hooks.test.ts

```diff
@@ -169,4 +169,140 @@ describe('ExpressInstrumentation hooks', () => {
       );
     });
   });
+
+  describe('span attributes hooks', () => {
+    let server: http.Server;
+    let port: number;
+    let rootSpan: Span;
+
+    beforeEach(async () => {
+      rootSpan = tracer.startSpan('rootSpan');
+
+      const httpServer = await serverWithMiddleware(tracer, rootSpan, app => {
+        app.get('*', (req, res) => {
+          res.send('ok');
+        });
+      });
+      server = httpServer.server;
+      port = httpServer.port;
+    });
+
+    afterEach(() => {
+      server.close();
+    });
+
+    it('should set span attributes', async () => {
+      instrumentation.setConfig({
+        spanAttributesHook: ({ request, route, layerType }, baseAttributes) => {
+          if (layerType) {
+            return {
+              route,
+            };
+          }
```

Comment:
Can this `if` statement be  `truthy` in the test? Are there any assertions to verify this code path?

Reply:
It can't be. I reused it from the name hook. I've removed it.